pkg/proxy/proxy.go

```diff
@@ -121,7 +124,7 @@ func (p *Proxy) StartProxy() *http.Server {
 
 	// routes
 	wg := router.Group("/apis/toolchain.dev.openshift.com/v1alpha1/workspaces")
-	wg.GET("/:workspace", handlers.HandleSpaceGetRequest(p.spaceLister))
+	wg.GET("/:workspace", handlers.HandleSpaceGetRequest(p.spaceLister, commoncluster.GetMemberClusters))
```

Comment:
should we use `p.getMembersFunc` here?

Reply:
Good point, but I believe that wuold make it more difficult to test in `HandleSpaceGetRequest`, since we are directly invoking `HandleSpaceGetRequest` without creating a `Proxy` instance. 

WDYT?

Reply:
yeah, I agree with you on having to pass the function to `HandleSpaceGetRequest`.

Btw, on this line, should we use `p.getMembersFunc` instead of `commoncluster.GetMemberClusters`? To my understanding, this way we are not forwarding the `GetMemberClustersFunc` provided in `NewProxy` to the handler, but defaulting to the one in `common/cluster` lib.

Basically, is the following change correct?
```suggestion
	wg.GET("/:workspace", handlers.HandleSpaceGetRequest(p.spaceLister, p.getMemberClusters))
```

Reply:
Ohh now I understand you you mean, yes we should pass that function indeed. Don in https://github.com/codeready-toolchain/registration-service/pull/387/commits/9095d1ccd4d3fefce7a242b3249b557e486ab354 

Thanks and sorry for the confusion.